### PR TITLE
Merge in the flutter gallery

### PR DESCRIPTION
**Pull request by rami-a**

### CODE REVIEW

File: gallery/LICENSE [hunk 1]
```diff
@@ -0,0 +1,324 @@
+// Copyright 2019 The Flutter team. All rights reserved.
```
Comment (RedBrogdon) [inline gallery/LICENSE]: We've got a license file for the repo, so this can be left out.

File: gallery/README.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
+# Flutter  Gallery
```
Comment (RedBrogdon) [inline gallery/README.md]: nit: there's an extra space here

File: gallery/README.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+Flutter Gallery is a resource to help developers evaluate and use Flutter.
+It is a collection of material design widgets, behaviors, and vignettes
+implemented with Flutter. We often get asked how one can see Flutter in
+action, and this gallery helps them see what Flutter provides and how it
```
Comment (RedBrogdon) [inline gallery/README.md]: "Them" doesn't have an antecedent in this sentence. Consider something like "...this gallery demonstrates what Flutter provides..." instead.

File: gallery/gallery/README.md [hunk 4]
```diff
@@ -0,0 +1,7 @@
+# gallery
```
Comment (RedBrogdon) [inline gallery/gallery/README.md]: If the README in /gallery is meant to be the one for the whole gallery project, I'd remove this one.

Comment (rami-a) [reply gallery/gallery/README.md]: You're right, removed

File: gallery/README.md (hunk 3 above)
Comment (rami-a) [reply gallery/README.md]: good call, updated

File: gallery/README.md (hunk 2 above)
Comment (rami-a) [reply gallery/README.md]: Fixed

File: gallery/LICENSE (hunk 1 above)
Comment (rami-a) [reply gallery/LICENSE]: Removed

**Pull request merged**